src/components/SharedComponents/Buttons/AddObsButton.js

```diff
@@ -1,17 +1,34 @@
 // @flow
 
 import AddObsModal from "components/AddObsModal";
+import { INatIcon } from "components/SharedComponents";
 import Modal from "components/SharedComponents/Modal";
+import { View } from "components/styledComponents";
 import { t } from "i18next";
 import * as React from "react";
-import { IconButton, useTheme } from "react-native-paper";
+import { Pressable, StyleSheet } from "react-native";
+import LinearGradient from "react-native-linear-gradient";
+import colors from "styles/tailwindColors";
 
-const AddObsButton = ( ): React.Node => {
-  const theme = useTheme( );
+const shadow = StyleSheet.create( {
```

Comment:
Is it possible to use the nativewind shadow classname here instead. The nativewind shadow adds a box-shadow though, I don't know if that is any different than a "drop-shadow".
If not, at least the shadowColor should be black here.

Reply:
Unfortunately no, I think react-native does something special under the hood to apply the shadow effect. In any case, android specifically uses `elevation` style to determine the drop shadow effect.

Reply:
Great idea to have a central place for the drop-shadow style then. I have just moved it into the styles folder, that makes more sense to me.